### PR TITLE
MVP Complete

### PR DESCRIPTION
**Pull request by BarbosaRicardo**

### CODE REVIEW

File: Article/Article.js [hunk 1]
```diff
@@ -3,20 +3,23 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.domElement.querySelector('.article .expandButton');
```
Comment (IceyMint) [inline Article/Article.js]: you dont need to put .article in the query selector

**Review by IceyMint — APPROVED** [review]

File: LESS/index.less [hunk 2]
```diff
@@ -4,6 +4,8 @@
 /* Below this line is where you will start your work */
 /* It is important to note, you will not be writing any CSS in this file*/
 @import '../Header/Header.less';
+@import '../Menu/Menu.less';
```
Comment (jay-maas) [inline LESS/index.less:7]: 👍

File: Menu/Menu.js [hunk 3]
```diff
@@ -1,10 +1,56 @@
...
+  return menu;
 }
 
-// Start Here: Create a reference to the ".menu" class
-const menu;
-// create a reference to the ".menu-button" class
-const menuButton;
-// Using your menuButton reference, add a click handler that calls toggleMenu
+const header = document.querySelector('.header');
+header.appendChild(menuComponent(menuItems));
```
Comment (jay-maas) [inline Menu/Menu.js:56]: 👍

File: Menu/Menu.js [hunk 4]
```diff
@@ -1,10 +1,56 @@
...
+  arr.forEach(item => {
+    let menuItem = document.createElement('li');
+    menuItem.textContent = item;
+
+    menuList.appendChild(menuItem);
+  });
+
+  const menuButton = document.querySelector('.menu-button');
+
+  menuButton.addEventListener('click', () => {
```
Comment (jay-maas) [inline Menu/Menu.js:48]: 👍

File: LESS/index.less [hunk 5]
```diff
@@ -4,6 +4,8 @@
...
+@import '../Article/Article.less';
```
Comment (jay-maas) [inline LESS/index.less:8]: 🌮

File: Article/Article.js [hunk 6]
```diff
@@ -1,29 +1,160 @@
...
+
+  //set class names 
+  article.classList.add('article')
+  articleDate.classList.add('date')
+  expand.classList.add('expandButton')
+  
+
+  //span event
+  expand.addEventListener('click', () => {
+    article.classList.toggle('article-open')
```
Comment (jay-maas) [inline Article/Article.js:153]: 🌮

File: index.html [hunk 7]
```diff
@@ -111,6 +111,25 @@ <h2>Professional Software Development in 2018</h2>
...
-    </div>
+      <div class ="article">
+        <h2>Danearyus' High Valaryian Speech</h2>
+        <p class = "date">June 10, 2019</p>
+        <p>Valor dohaeris. Bantis zobrie issa se ossyngnoti ledys. Avy jorraelan. 
+          Skoros morghot vestri? Vilibazmosa iderenni emilun. Skoros morghot vestri? 
+          Ao ynoma diniluks? Skorī demalyti tymptir tymis, erinis ia morghulis. </p> 
+          <p>Valar morghulis. Zyhys onoso jehikagon Aeksiot epi, se gis hen syndrorro jemagon. 
+            Zyhys perzys stepagon Aeksio Ono jorepi, se morghultas lys qelitsos sikagon. 
+            Zyhys perzys stepagon Aeksio Ono jorepi, se morghultas lys qelitsos sikagon.
```
Comment (jay-maas) [inline index.html:122]: 🌮

**Review by jay-maas — APPROVED** [review]
🌮 Fantastico! Que rico!

File: LESS/index.less (hunk 2 above)
Comment (BarbosaRicardo) [reply LESS/index.less:7]: Why one taco? @jay-maas

Comment (jay-maas) [reply LESS/index.less:7]: 🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮🌮

Comment (jay-maas) [reply LESS/index.less:7]: hope your hungry :)

**Pull request merged**